dotcom-rendering/cypress/lib/mockDate.js

```diff
@@ -0,0 +1,19 @@
+export class MorningDate extends Date {
+	constructor(date) {
+		super(date);
+		if (date) {
+			return new Date(date);
+		}
+		return new Date(2023, 9, 9, 9, 30, 0);
+	}
+}
+
+export class AfternoonDate extends Date {
+	constructor(date) {
+		super(date);
+		if (date) {
+			return new Date(date);
+		}
+		return new Date(2023, 9, 9, 17, 30, 0);
+	}
+}
```

Comment:
Could we use [Cypress’s built-in `clock` method](https://docs.cypress.io/api/commands/clock)?